public/javascripts/payments.js

```diff
@@ -712,6 +830,24 @@
     showRelevantPaymentMethods();
   };
 
+  const selectCurrency = (currency) => {
+    const selector = document.getElementById('currency');
+    selector.querySelector(`option[value=${currency}]`).selected = 'selected';
+    selector.className = `field ${country.toLowerCase()}`;
```

Comment:
[classList](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList) [support](https://caniuse.com/#feat=classlist) is pretty good now.